docs/architecture/decision-records/GithubActionWorkflow.md

```diff
@@ -0,0 +1,23 @@
+# GitHub Action (GHA) Reusable Workflows
+
+## Context
+
+The main GHA workflow file (continuous-deployment.yml) had grown in size and complexity. To improve readability and maintainability job steps will be grouped into reusable workflows. To use a GHA reusable workflow the yaml file **must** reside directly in the path `.github/workflows`. Therefore, subfolders cannot be used to organize files in this path.
```

Comment:
I would prefer wrapping "continuous-deployment.yml" in "`" rather than parentheses. I also might add to the first sentence something like:
> The main GHA workflow file `continuous-deployment.yml` had grown in size and complexity, making changes difficult for someone unfamiliar with the details.